content/en/docs/setup/getting-started/index.md

```diff
@@ -68,146 +52,333 @@ Download the Istio release which includes installation files, samples, and the
+but rather a host name, and the above command will have failed to set the `INGRESS_HOST` environment variable.
+Use the following command to correct the `INGRESS_HOST` value:
+
+{{< text bash >}}
+$ export INGRESS_HOST=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].hostname}')
+{{< /text >}}
+
 {{< /warning >}}
 
+**Follow these instructions if your environment does not have an external load balancer and choose a node port instead.**
+
+Set the ingress ports:
+
+{{< text bash >}}
+$ export INGRESS_PORT=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.spec.ports[?(@.name=="http2")].nodePort}')
+$ export SECURE_INGRESS_PORT=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.spec.ports[?(@.name=="https")].nodePort}')
+{{< /text >}}
+
+_GKE:_
+
 {{< text bash >}}
-$ kubectl label namespace <namespace> istio-injection=enabled
+$ export INGRESS_HOST=<workerNodeAddress>
 {{< /text >}}
 
-Alternatively, you can manually inject Envoy containers in your application pods before deploying them, using
-[`istioctl kube-inject`](/docs/reference/commands/istioctl/#istioctl-kube-inject):
+You need to create firewall rules to allow the TCP traffic to the `ingressgateway` service's ports.
+Run the following commands to allow the traffic for the HTTP port, the secure port (HTTPS) or both:
 
 {{< text bash >}}
-$ istioctl kube-inject -f <your-app-spec>.yaml | kubectl apply -f -
+$ gcloud compute firewall-rules create allow-gateway-http --allow tcp:$INGRESS_PORT
+$ gcloud compute firewall-rules create allow-gateway-https --allow tcp:$SECURE_INGRESS_PORT
 {{< /text >}}
 
-## Deploy an application or the Bookinfo sample {#Bookinfo}
+_Docker For Desktop:_
 
-If you have an application ready to go, deploy it:
+{{< text bash >}}
+$ export INGRESS_HOST=127.0.0.1
+{{< /text >}}
+
+_Other environments (e.g., IBM Cloud Private, etc.):_
 
 {{< text bash >}}
-$ kubectl create -n <namespace> -f <your-app-spec>.yaml
+$ export INGRESS_HOST=$(kubectl get po -l istio=ingressgateway -n istio-system -o jsonpath='{.items[0].status.hostIP}')
 {{< /text >}}
 
-Alternatively,
-[deploy the Bookinfo sample](/docs/examples/bookinfo/)
-to evaluate Istio's features for traffic routing,
-fault injection, rate limiting, etc. Then, explore the various
-[Istio tasks](/docs/tasks/) that interest you.
+{{< /tab >}}
+
+{{< /tabset >}}
+
+1.  Set `GATEWAY_URL`:
+
+    {{< text bash >}}
+    $ export GATEWAY_URL=$INGRESS_HOST:$INGRESS_PORT
+    {{< /text >}}
+
+1.  Ensure an IP address and port were successfully assigned to the environment variable:
+
+    {{< text bash >}}
+    $ echo $GATEWAY_URL
+    192.168.99.100:32194
+    {{< /text >}}
+
+### Verify external access {#confirm}
 
-## Next evaluation steps
+Confirm that the Bookinfo application is accessible from outside the cluster by
+navigating in your browser to `http://<your $GATEWAY_URL value>/productpage`.
```

Comment:
Put this command in a text bash.

Reply:
This isn't a bash command. But is there a way to open the GUI browser to a specific URL using a bash command?

Reply:
I think you can just use text. I incorrectly had bash in my prior remark.

Reply:
fixed